src/native-common/utils/EventHelpers.ts

```diff
@@ -59,6 +59,12 @@ export class EventHelpers {
                         keyCode = 18;
                         break;
 
+                    //keyCode in windows is 93 for context menu button. Since that is already used 
```

Comment:
Nit: Please add spaces after "//" for consistency.

Reply:
Added in all comments